src/threading/formatters/JSON.cc

```diff
@@ -25,80 +25,85 @@ JSON::~JSON()
 	}
 
 bool JSON::Describe(ODesc* desc, int num_fields, const Field* const * fields,
-                    Value** vals) const
+					Value** vals) const
```

Comment:
Just a small note - this change actually breaks formatting that was correct before (will fix during merge).